packages/candy_editor/package.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "@dapp/candy-editor",
+    "version": "0.1.0",
+    "description": "Candy Editor used within the dApps.",
```

Comment:
Let's give this a better description like:
"Candy Editor for building, editing and viewing candy data."